conf/nginx.conf

```
@@ -8,6 +8,9 @@ location / {
   if ($scheme = http) {
     rewrite ^ https://$server_name$request_uri? permanent;
   }
+
+  client_max_body_size 0;
```

```suggestion
  # Common parameter to increase upload size limit in conjunction with dedicated php-fpm file
  client_max_body_size 50M;
```
I propose to use default example_ynh value

---

Why not, but the idea is to benefit of the configuration available on wikijs admin interface. Wikijs offer the possibility to define the "max body size" in admin interface, it could be great to let wikijs manage this behavior to not create some confusion for users (that will open tickets) or will do this modification.